pkg/crypto/clef.go

```diff
@@ -0,0 +1,91 @@
+package crypto
```

Comment:
It looks to me that clef signer implementation can be in its own package and to drop Clef in all identifiers here. Also clef imports go-ethereum, and if somewhere crypto package is imported for anything other than clef, go-ethereum would be an unused dependency.

Reply:
created a clef package and moved everything there